docs/variants/msi_z790/releases.md

```diff
@@ -68,3 +175,11 @@ subscription via Dasharo Entry Subscription newsletter.
 - [Intel microcode based on RPL B0 0x00000113 revision microcode-20230613](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/tree/microcode-20230613/intel-ucode/06-b7-01)
 
 [newsletter]: https://newsletter.3mdeb.com/subscription/KgJ7V_mmJ
+[msi_ms7e06_v0.9.1_ddr4.rom_hash]: https://dl.3mdeb.com/open-source-firmware/Dasharo/msi_ms7e06/v0.9.1/msi_ms7e06_v0.9.1_ddr4.rom.sha256
+[msi_ms7e06_v0.9.1_ddr4.rom_sig]: https://dl.3mdeb.com/open-source-firmware/Dasharo/msi_ms7e06/v0.9.1/msi_ms7e06_v0.9.1_ddr4.rom.sha256.sig
+[msi_ms7e06_v0.9.1_ddr4_dev_signed.rom_hash]: https://dl.3mdeb.com/open-source-firmware/Dasharo/msi_ms7e06/v0.9.1/msi_ms7e06_v0.9.1_ddr4_dev_signed.rom.sha256
+[msi_ms7e06_v0.9.1_ddr4_dev_signed.rom_sig]: https://dl.3mdeb.com/open-source-firmware/Dasharo/msi_ms7e06/v0.9.1/msi_ms7e06_v0.9.1_ddr4_dev_signed.rom.sha256.sig
```

Comment:
ddr5 not ddr4

Reply:
> But I am surprised we are publishing binaries v0.9.1

We are not. It is hash and signature, so reproducibility can be checked for instance.

Reply:
@macpijan You are right. I did check the links in the newsletter, so I figured these are the same. Great, thank you for pointing this out. 

> We didn't test Z790 with DDR4 at all. So let's just add this disclaimer, to the newsletter, just to be clear for users. And it will address my concerns in this aspect.

 We should still make it clearer in a newsletter, though, @miczyg1 but not here. I am resolving this thread.